docs/source/advanced/cmake.rst

```diff
@@ -174,6 +174,41 @@ alpaka_OFFLOAD_MAX_BLOCK_SIZE
      Maximum number threads per block to be suggested by any target offloading backends
      ANY_BT_OMP5 and ANY_BT_OACC.
 
+
+CMAKE_CXX_FLAGS
+  Target architecture and some compiler specific flags have to be set manually:
+  
+  * Clang / AOMP / rocmClang
+    
+    * x86: `-fopenmp -fopenmp-targets=x86_64-pc-linux-gnu`
+      
+    * ppc64le: `-fopenmp -fopenmp-targets=ppc64le-pc-linux-gnu`
+      
+    * hsa: `-fopenmp -fopenmp-targets=amdgcn-amd-amdhsa -Xopenmp-target=amdgcn-amd-amdhsa -march=<arch>`
+      
+      A list of AMD GPU architectures can be found `here <https://llvm.org/docs/AMDGPUUsage.html#processors>`_.
+      
+    * nvptx: `-fopenmp -fopenmp-targets=nvptx64-nvidia-cuda`
+
+    CMake actually sets `-fopenmp` automatically, but OpenMP support is
+    detected after the deteced compiler is tested, which fails with
+    `-fopenmp-targets=` being present alone.
+      
+  * GCC
+    
+    * host: `-foffload=disable`
+      
+      At run time set the environment variable `OMP_TARGET_OFFLOAD=DISABLED`
+      
+    * nvptx: `-foffload=nvptx-none`
+      
+  * NVHPC
+    
+    * host: `-ta=host` or `-ta=-ta=multicore`
```

Comment:
```suggestion
    * host: `-ta=host` or `-ta=-ta=multicore`
```

The multicore part seems like a duplication to me.

Reply:
It is not. "host" means single-thread.

Reply:
Yes, but is `-ta=-ta=multicore` correct?

Reply:
I did not notice this, sorry. The second `-ta=` is wrong. You suggested change does not change anything, though.

Reply:
```suggestion
    * host: `-ta=host` or `-ta=multicore`
```